interop/test/ping.spec.ts

```diff
@@ -38,8 +40,11 @@ describe('ping test', () => {
     const MUXER = process.env.muxer
     const IP = process.env.ip ?? '0.0.0.0'
 
-    const options: Libp2pOptions<{ ping: PingService, identify: DefaultIdentifyService }> = {
+    const options: Libp2pOptions<{ ping: PingService, identify: unknown }> = {
```

Comment:
This has been [refactored](https://github.com/libp2p/js-libp2p/pull/1784/files#diff-024eabde3750f3bf282cd9a642bd16ba0357c8c0b3110ffa4c78bfdd60f9ed2fR27)

Reply:
Rebased 👍